.travis.yml

```diff
@@ -1,19 +1,27 @@
 language: python
 matrix:
   include:
-    - name: "Python 3.6"
-      python: 3.6
-    - name: "Python 3.7"
-      python: 3.7
-install: 
-  - echo "Install starting"
-  - pip3 install ".[dev]"
-  - echo "Install ending"
-
+  - name: Python 3.6
+    python: 3.6
+  - name: Python 3.7
+    python: 3.7
+install:
+- echo "Install starting"
+- pip3 install ".[dev]"
+- echo "Install ending"
 script:
-  - echo "Script starting"
-  - flake8 --config flake8.cfg battenberg
-  - pytest --cov=battenberg
-  # Upload the code coverage report to codecov
-  - codecov
-  - echo "Script ending"
+- echo "Script starting"
+- flake8 --config flake8.cfg battenberg
+- pytest --cov=battenberg
+- codecov
+- echo "Script ending"
+deploy:
+  provider: pypi
+  user: __token__
+  password:
+    # This is an encrypted copy of the PyPI API token which you get from your PyPI Account Settings page.
+    # Generated using: travis encrypt <pypi api token> --add deploy.password
+    secure: HH8cRveKVVLAlSnnKARV6SuX9+VZi5tfaaM/r8lXxyaM2gCf6i2tPuuagmGhW8Zv0mPR/8qn21s2/hHIo55M2c2l3/WbLwGzeMv57aObvvHDDRfgiHIMEyDS5jhPyqpW+lJkzuwUDjimpa+f8qGQVqxTVjg7PZoJ8/twYRP4tZ8CZaIa9RG9xqvVjoYzt2mvrlnUcW28GIZYJ+BAy5TPZAFTmsmR41aIiLSZu3BTNLadydbQUVKKkIxxvVW2VwGaUmt8QkG3Cs+UPS0N8jWAkc4hafL1Yly+dMvEHDuqELxjW1cDDEeLY35m5WGdA1nxxd8hQjId3rd6gaVdyxjFOKScEChR7Hp/B+ktDbimGOkJSVAGrPYpCuXRwvwEWRA9Sn9vqSdVNzp7AgYyDjSyGRfqer2iPqo+JjUwisF1WLDz77ZmFWrmLrUtbDgtE/CM5LK2lY/ixIO7hZ+e2wRH4V7ifvBCHYItDMFWdLcGy9VVu4epAr3CsmzoqQdb587uoi+rV6wEwkCPHkn2mDiqgGVVjE3A9VwKcSJYmLIHQEFDtp6UgieJdBM023LXbNPjMlUcG1HGVzVXNNTuPY4qudogftXJJ84n9ZqjINOr++PMaor7Vm+bCUnZ+d6XvTbypZnFs3GKoPseZ8kJs9xSlP1fZLGA3Z5v+TgiWB7XKPg=
```

Comment:
hm...

Reply:
I'm afraid so, I couldn't work out a way (in my 2 mins of Googling) how to set up publishing to the global PyPI as an organization. Any ideas?

Reply:
Not sure, but @toumorokoshi  has published some eggs on PyPI. Maybe he knows the best practice here.

Reply:
can i use this token now to publish artifacts from my own repo in github as @alexlatchford ?

Reply:
I would assume not as I think Travis is doing some decryption magic under the hood to make this work @apryiomka

Reply:
Looks like Zillow does have an account (https://pypi.org/user/zillow/), just acquired an API token and updated the PR to include it :)